quickstarts/microsoft.azurestackhci/create-cluster-2-node-switched-custom-storageip/azuredeploy.parameters.json

```diff
@@ -3,7 +3,7 @@
     "contentVersion": "1.0.0.0",
     "parameters": {
         "apiVersion": {
-            "value": "2024-04-01"
+            "value": "2024-09-01-preview"
```

Comment:
API Versions should not be parameterized. Can this (and other instances of this) parameter be removed, and the API version can be hardcoded?

Reply:
We have used API version in the parameters to allow extensibility to the customers.
If we remove this api version now, this would mean a breaking change for consumers at this point.
Would be to possible to allow us take in this in a planned manner in future iterations ?